packages/components/src/search-control/index.tsx

```diff
@@ -53,7 +55,9 @@ function SearchControl(
 					label={ __( 'Reset search' ) }
 					onClick={ () => {
 						onChange( '' );
-						searchRef.current.focus();
+						if ( searchRef.current ) {
```

Comment:
Without this `if` conditional, there's a TypeScript error:

```
(property) React.MutableRefObject<undefined>.current: undefined
Conversion of type 'undefined' to type 'HTMLInputElement' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.ts(2352)
```

Reply:
You do not need it if you do this https://github.com/WordPress/gutenberg/pull/42647#discussion_r928627036

Reply:
Thanks! Removed this conditional in https://github.com/WordPress/gutenberg/pull/42647/commits/366b312db28c55cf9694b5021bca421c6065196b